nautobot/docs/release-notes/version-1.5.md

```diff
@@ -148,6 +148,24 @@ A number of mixin classes have been renamed and/or relocated for improved self-c
 | `TenancyFilterSet`             | `TenancyModelFilterSetMixin`                 |
 
 <!-- towncrier release notes start -->
+## v1.5.20 (2023-05-30)
+
+### Added
+
+- [#na](https://github.com/nautobot/nautobot/issues/na) - Add troubleshooting steps for Azure AD SSO Group Sync example.
+- [#3400](https://github.com/nautobot/nautobot/issues/3400) - Added documentation on how to enable Jobs and Job hooks.
```

Comment:
```suggestion
- [#3400](https://github.com/nautobot/nautobot/issues/3400) - Added documentation on how to enable Jobs and Job hooks.
- [#3766](https://github.com/nautobot/nautobot/issues/3766) - Add troubleshooting steps for Azure AD SSO Group Sync example.
```

Reply:
Interesting... Musta missed this in the original review.